package.json

```diff
@@ -21,12 +21,12 @@
   "author": "Roman Shtylman <shtylman@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "mime": "1.2.11",
     "browserify": "4.1.9",
-    "filewatcher": "1.1.1",
-    "uglify-js": "2.4.13",
+    "convert-source-map": "1.1.1",
```

Comment:
it's probably from `npm --save`. it sorts dependencies alphabetically so that you don't run into the annoying diff problem. :-)